geniza/corpus/templates/corpus/document_list.html

```diff
@@ -5,30 +5,33 @@
 {% block meta_description %}{{ page_description }}{% endblock meta_description %}
 
 {% block main %}
-    <h1>{{ page_title }}</h1>
```

Comment:
I almost missed that you removed this! I know there is no title in Gissoo's designs (which now I'm kind of questioning a little bit); I think it's still valuable for screen readers, and maybe search engines. I propose we preserve the page-level h1 and make it screen-reader only.